xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-6.30.0
+6.31.0
```

Comment:
@InuSasha v17 alpha 1 has been tagged, SO v17 PRs started being merged.

A rebase and a jsonrpc bump maybe a good idea.https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/version.txt to 6.34.0